x-pack/solutions/search/plugins/search_homepage/public/components/ai_search_capabilities/keyword_search.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+
+import { AISearchWorkflow } from './ai_search_workflow';
+import { useAssetBasePath } from '../../hooks/use_asset_base_path';
+
+export const KeywordSearch: React.FC = () => {
+  const assetBasePath = useAssetBasePath();
+
+  const keywordSearchText = {
+    image: `${assetBasePath}/keyword_search.svg`,
+    imageAlt: i18n.translate('xpack.searchHomepage.aiSearchCapabilities.keywordSearch.imageAlt', {
+      defaultMessage: 'Keyword Search',
+    }),
+    heading: i18n.translate('xpack.searchHomepage.aiSearchCapabilities.keywordSearch.title', {
+      defaultMessage: 'Setup your application with Elasticsearch’s full-text search capabilities',
```

Comment:
>      defaultMessage: 'Setup your application with Elasticsearch’s full-text search capabilities',

IMO this is another scenario where the text makes me think it's going to walk me through a longer workflow of creating an app and learning to search but it seems like I only get led through the creation of a simple index. If that's an accurate assessment of the current limited workflow, I think it would be helpful to link to https://www.elastic.co/docs/solutions/search/full-text and make it clear that there's a longer journey ahead than just clicking the button.